src/transformers/__init__.py

```diff
@@ -5441,6 +5450,7 @@
     )
     from .models.phi import PhiConfig
     from .models.phi3 import Phi3Config
+    from .models.phimoe import PhiMoEConfig
```

Comment:
```suggestion
    from .models.phimoe import PhimoeConfig
```
let's follow `OlmoeForCausalLM` convention of camel cased

Reply:
Fixed